README.md

```diff
@@ -3,7 +3,7 @@
 <a name="readme-top"></a>
 
 <div align="center"> 
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Kindly remove the microverse logo as suggested. Currently its a broken image so remove it.